### PR TITLE
Remove Extra NewLines between Comments and Inputs/Outputs to fix MarkDownLint Warnings

### PR DESCRIPTION
**Pull request by X-Guardian**
### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

For more information, see the [Contributing Guide](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Currently, if a main.tf has comments and the module has inputs and/or outputs, an additional newline is added between the comments and the inputs/outputs in the markdown. This is reported by [MarkDownLint](https://github.com/markdownlint/markdownlint) as a warning:
```
MD012/no-multiple-blanks: Multiple consecutive blank lines [Expected: 1; Actual: 2]
```
This pull request removes those additional newlines.

### Checklist

Put an `x` into all boxes that apply:

- [x] I have read the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

#### Tests

- [x] I have modified tests to cover my changes.
- [x] All tests pass when I run `make test`.

I've modified the markdown golden test files, but the markdown tests fail for me, but I think it is because I am running them on Windows and it is a \r\n issue. Can someone confirm?

#### Documentation

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

#### Code Style

- [x] My code follows the code style of this project.

### CODE REVIEW

File: internal/pkg/print/markdown/markdown.go [hunk 1]
```diff
@@ -27,6 +27,9 @@ func Print(document *doc.Doc, settings settings.Settings) (string, error) {
 	}
 
 	if document.HasOutputs() {
+		if document.HasInputs() {
```
Comment (metmajer) [inline internal/pkg/print/markdown/markdown.go]: Please move this block down between the sorting and the call to `printOutputs`.

**Comment by metmajer** [comment]
@X-Guardian thanks for providing this fix 👍 Have you run `make test`?

File: internal/pkg/print/markdown/markdown.go (hunk 1 above)
Comment (X-Guardian) [reply internal/pkg/print/markdown/markdown.go]: Done.

**Comment by X-Guardian** [comment]
@metmajer, the markdown tests fail for me, but I think it is because I am running them on Windows and it is a \r\n issue. Can you confirm?

**Comment by metmajer** [comment]
@X-Guardian I've run the tests on my machine and some tests fail. Good news, there's a pattern: the comment in `main.tf`s header includes an extra (unexpected) newline.

This reminds me that CircleCI should actually run a pipeline that shows the test execution to you in this pull request. I'll try to have this fixed.

**Comment by metmajer** [comment]
@X-Guardian CircleCI should now run a pipeline when you add to this pull request. Want to give it a try?

**Comment by X-Guardian** [comment]
Cheers @metmajer, I added another commit, but no CircleCI was triggered...

**Comment by X-Guardian** [comment]
@metmajer, I wonder if your CircleCI change will only trigger on a new pull request? Shall I create one to test?

**Comment by metmajer** [comment]
@X-Guardian I haven't thought about this, while I would strongly assume that the pipeline is triggered by a commit, it's worth a try. Thanks!

**Comment by metmajer** [comment]
@X-Guardian we've been able to fix the missing CircleCI builds for pull requests in #71. If you take a look, you should now an indicator (a green check mark or a red cross) indicating the status of your commit. By clicking on the indicator, you'll see an overlay where you can inspect the stages of the build. Beware that, for looking at the tests, I had to scroll down in the overlay.

**Comment by X-Guardian** [comment]
@metmajer, CircleCI now working on commits, and I've fixed all the spurious blank lines from the tests.

**Comment by metmajer** [comment]
@X-Guardian thanks for staying on it. Appreciate your contribution!

**Pull request merged**